WordPressVIPMinimum/Sniffs/Files/IncludingFileSniff.php

```diff
@@ -163,13 +184,35 @@ public function process_token( $stackPtr ) {
 			return;
 		}
 
-		if ( $this->tokens[ $nextToken ]['code'] === T_CONSTANT_ENCAPSED_STRING && filter_var( str_replace( [ '"', "'" ], '', $this->tokens[ $nextToken ]['content'] ), FILTER_VALIDATE_URL ) ) {
-			$message = 'Include path must be local file source, external URLs are prohibited on WordPress VIP.';
-			$this->phpcsFile->addError( $message, $nextToken, 'ExternalURL' );
-			return;
-		}
-
 		$message = 'Absolute include path must be used. Use `get_template_directory()`, `get_stylesheet_directory()` or `plugin_dir_path()`.';
 		$this->phpcsFile->addError( $message, $nextToken, 'NotAbsolutePath' );
 	}
+
+	/**
+	 * Check if a content string contains wording found in custom paths.
+	 *
+	 * @param int    $stackPtr Optional. The position of the current token in the
+	 *                         token stack.
+	 *                         This parameter needs to be passed if no $content is
+	 *                         passed.
+	 * @param string $content  Optionally, the current content string, might be a
+	 *                         substring of the original string.
+	 *                         Defaults to `false` to distinguish between a passed
+	 *                         empty string and not passing the $content string.
+	 *
+	 * @return bool True if the string contains a keyword in $customPaths, false otherwise.
+	 */
+	public function has_custom_path( $stackPtr = null, $content = null ) {
```

Comment:
* Why is this a `public` method ?
* Why are both parameters optional ?
* Why is no check being done against the token code/type ? This check will now also bail out on a call to `dirname()` or on the variable `$custom_path` which is not the intention of this check.